Samples/FilterOnObservableCondition/MainView.js

```diff
@@ -35,11 +35,11 @@ var items = Observable(
 	new Color('#FFFF00', 'Other 5', COLOR_TYPES.OTHER)
 );
 
-var filteredItems = currentCategory.map(function(category){
```

Comment:
Then the Observable docs are outdated.

For single-value Observables, `map(fn).inner()` is equivalent `flatMap(fn)`.
I've also verified that the example behaves the same.